src/main/groovy/org/mockito/release/notes/contributors/RecentContributorsFetcher.java

```diff
@@ -0,0 +1,118 @@
+package org.mockito.release.notes.contributors;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.json.simple.DeserializationException;
+import org.json.simple.JsonObject;
+import org.mockito.release.notes.model.Contributor;
+import org.mockito.release.notes.util.GitHubListFetcher;
+
+import java.io.IOException;
+import java.util.*;
+
+import static org.mockito.release.internal.util.ArgumentValidation.notNull;
+import static org.mockito.release.internal.util.DateUtil.forGitHub;
+import static org.mockito.release.internal.util.DateUtil.yesterday;
+
+/**
+ * Fetches recent contributors from GitHub using the "commit" end point.
+ * The "contributors" endpoint does not return the most recent contributors as documented.
+ */
+public class RecentContributorsFetcher {
+
+    private static final Logger LOG = Logging.getLogger(RecentContributorsFetcher.class);
+
+    /**
+     * Contributors that pushed commits to the repo withing the last 24hrs
+     */
+    public Collection<Contributor> fetchContributorsSinceYesterday(String repository, String readOnlyAuthToken) {
+        String yesterday = forGitHub(yesterday());
```

Comment:
24h worked for me :) We'll see if notes have right stuff. We can always come back and make it 48h. GitHub docs say contributors endpoint is 4h delayed so 24 should be ok.